src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -473,6 +473,14 @@ impl Step for Rustc {
                     );
                 }
             }
+
+            {
+                let src_dir = builder.sysroot_libdir(compiler, host).parent().unwrap().join("bin");
```

Comment:
Remark: this seems like it should be `sysroot_bindir` if it doesn't already exist

Reply:
Makes sense, fixed.